changelog/_unreleased/2025-01-22-send-small-mail-object-directly.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Send small mail object directly without filesystem access
+author: Benjamin Wittwer
+author_email: benjamin.wittwer@a-k-f.de
+author_github: akf-bw
+---
+# Core
+* Changed `Shopware\Core\Content\Mail\Service\MailSender` to directly send a mail to the mailer without using the filesystem
```

Comment:
we should mention here, that this only happens if the size is a specific value. otherwise this could be read wrong